### PR TITLE
Add *.yml config file to /etc/sysconfig when packaging

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag @jesuspg 
#### Description

Main configuration file is now at `/etc/sysconfig` after installing the package, and subdirectory `config/` only includes a sample file. When package is upgraded, `yum` will create a backup of such configuration file.
#### Testing

Verified on CentOS 6.3

### CODE REVIEW

File: dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard [hunk 1]
```diff
@@ -31,17 +31,17 @@ LOGFILE=/var/log/$NAME/$NAME.log
...
-
-# Check that networking is up
 [ -r /etc/sysconfig/network ]	&& . /etc/sysconfig/network
 [ "${NETWORKING}" = "no" ]	&& exit 0
 
 # Source function library
 . /etc/rc.d/init.d/functions
 
+# Service configuration file
+[ -r /etc/sysconfig/$NAME.yml ]	&& DAEMON_ARGS="-c/etc/sysconfig/$NAME.yml"
```
Comment (flopezag) [inline dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard:43]: I suggest to leave a blank space after and before "&&" operator.

File: dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec [hunk 2]
```diff
@@ -201,9 +207,6 @@ if [ $1 -eq 0 ]; then
 fi
 
 %changelog
-* Mon Mar 03 2016 Telefónica I+D <opensource@tid.es> 1.4.1-1
```
Comment (flopezag) [inline dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec:204]: Are you sure that you want to delete this line?

Comment (pratid) [inline dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec:204]: Yes: deleted on purpose.

File: dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard (hunk 1 above)
Comment (pratid) [inline dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard:43]: Indentation with tabs is used for readability of these shell scripts.

**Comment by jesuspg** [comment]
LGTM

File: dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard (hunk 1 above)
Comment (flopezag) [inline dashboard/tools/build/files/redhat/SOURCES/etc/init.d/fihealth_dashboard:43]: NTC

File: dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec (hunk 2 above)
Comment (flopezag) [inline dashboard/tools/build/files/redhat/SPECS/fiware-fihealth-dashboard.spec:204]: Talking outside github.

NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**